WorldModel/WorldModel/Core/Languages/English.aslx

```diff
@@ -509,13 +509,10 @@
   <dynamictemplate name="DevModeErrorCantFindObject">"The object with the name '" + text + "' cannot be found."</dynamictemplate>
   <dynamictemplate name="DevModeErrorCantFindAttribute">"The attribute with the name '" + text + "' cannot be found."</dynamictemplate>
 
-  <!-- Need a special verb in English because of the name collision with the "enter" script rooms use -->
-  <verb name="enter_verb">
-    <pattern>enter #object#</pattern>
-    <property>enterverb</property>

```

Comment:
Will this be a breaking change for people upgrading, as `enterverb` is now `enter_verb`? Or does it not matter because this wasn't working in the first place?

Reply:
Oh, I thought I fixed that back to `enterverb`! I see I failed to push the commit... One sec...

Reply:
Done. Good catch! GitHub Desktop was just sitting there, waiting for me to push that commit.